embedchain/data_formatter/data_formatter.py

```diff
@@ -32,6 +33,20 @@ def _lazy_load(self, module_path: str):
         module = import_module(module_path)
         return getattr(module, class_name)
 
+    def _get_default_loader_config(self, type: DataType):
+        custom_loader_default_config = {
+            DataType.POSTGRES: DEFAULT_POSTGRES_CONFIG,
+        }
+
```

Comment:
Don't need this check since we want the user to provide the postgres config. If not passed, then error out.

Reply:
Cool, but still we need to separate the data types for which we need users to pass the loader. Updating the code accordingly